<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prescription</class>
 <widget class="QWidget" name="Prescription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Insert prescription data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_patient_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient data</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_drug_conc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Drug and concentration, quantity, dose, refund</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_drug">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_ID">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_ID">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Generated ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_return">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_add">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_add_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to list</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
